downloads/verification-notes/release-artfiacts-verification.md

```diff
@@ -1,64 +1,75 @@
-# Official Ballerina {{ version }} Release Artifacts
+# Official Ballerina {{ version }} release artifacts
 
 
-## Linux
+## Signatures and verification
 
-- [Linux-Deb-Installer](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.deb)  - [Linux-Deb-Certificate](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.deb.pem)  - [Linux-Deb-Signature](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.deb.sig)
+Ballerina uses [`Sigstore/cosign`](https://github.com/sigstore/cosign) for signing and verifying the release artifacts. The release artifacts are listed below:
 
-- [Linux-Rpm-Installer](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.rpm)  - [Linux-Rpm-Certificate](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.rpm.pem)  - [Linux-Rpm-Signature](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.rpm.sig)
 
+### Linux
 
-## MacOS
+The artifacts listed below are the Linux installers along with their respective certificate and signature files:
 
-- [MacOS-Installer](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-macos-x64.pkg)  - [MacOS-Certificate](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-macos-x64.pkg.pem)  - [MacOS-Signature](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-macos-x64.pkg.sig)
+- [Linux-Deb-Installer](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.deb)  - [Linux-Deb-Certificate](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.deb.pem)  - [Linux-Deb-Signature](https://github.com/ballerina-platform/ballerina-distribution/releases/download/v{{ version }}/ballerina-{{ version }}-swan-lake-linux-x64.deb.sig)
```

Comment:
Since the table format looks a bit congested, we decided to go with following format. I'v provided an example view of a run on my end for clarification.
![image](https://github.com/ballerina-platform/ballerina-dev-website/assets/85722530/6e646d95-d2f5-450d-9f62-29214703cbdd)

Reply:
As a  best practice of documentation, we shouldn't be adding many elements in one bullet point. If we have many, we need to add them in a tabular format.

Reply:
Okay ill rectify this